Modules/Blog/classes/class.ilBlogGlobalTemplate.php

```diff
@@ -0,0 +1,2059 @@
+<?php
+/* Copyright (c) 1998-2009 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+include_once("./Services/UICore/lib/html-it/IT.php");
+include_once("./Services/UICore/lib/html-it/ITX.php");
+
+/**
+* special template class to simplify handling of ITX/PEAR
+* @author	Stefan Kesseler <skesseler@databay.de>
+* @author	Sascha Hofmann <shofmann@databay.de>
+* @version	$Id$
+*/
+class ilBlogGlobalTemplate
```

Comment:
I'm not happy with the dozens of copies of the old `ilTemplate` class. They should at least inherit from one new (temporary) base class. Without this inheritance bugs have to be fixed in every new `il<Component>Template` class by the maintainer(?).

Reply:
Ok, sorry I am reading this comment right now, since I was on the "code" page the whole time...

So up to now there is now general reason not to extend ilGlobalTemplate but the maintainers should take care of this step? This is fine for me.

Reply:
I understand and already understood the intention of doing this. As long as the maintainers could decide whether or not extending from `ilGlobalTemplate` on their own (risk) and you don't forbid this by any rules, this is fine for me.